src/Uno.UI/UI/Xaml/Controls/Grid/DefinitionCollectionBase.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace Windows.UI.Xaml.Controls
+{
+	internal interface DefinitionCollectionBase : IList<DefinitionBase>
+	{
+		DefinitionBase GetItem(int index);
```

Comment:
Should be internal as well ?
```suggestion
		internal DefinitionBase GetItem(int index);
```